util/scale_run.sh

```diff
@@ -0,0 +1,230 @@
+#!/bin/bash
+
+# Run scale tests for each Standard Ref Arch size on docs page
+#   Ref: https://puppet.com/docs/pe/latest/hardware_requirements.html
+
+
+##########
+# Function Definitions
+##########
+
+# Display usage help
+usage () {
+    echo "$0 [-h] [-d] PE_VERSION -- Run scale testing on specified PE_VERSION"
+    echo
+    echo "where:"
+    echo "            -h  show this help text"
+    echo "            -d  debug mode: prints ENV values and commands to be executed, but does not run anything"
+    echo "    PE_VERSION  in semver notation (e.g 2019.1.0)"
+    echo
+}
+
+# Debug function to validate that bash subshells inherit
+# expected environment variables.
+function echo_env () {
+    echo "============================================
+In subshell for $test $run_type round $i
+cmd=$cmd
+BEAKER_INSTALL_TYPE=$BEAKER_INSTALL_TYPE
+BEAKER_PE_DIR=$BEAKER_PE_DIR
+BEAKER_PE_VER=$BEAKER_PE_VER
+PUPPET_GATLING_SCALE_BASE_INSTANCES=$PUPPET_GATLING_SCALE_BASE_INSTANCES
+PUPPET_GATLING_SCALE_ITERATIONS=$PUPPET_GATLING_SCALE_ITERATIONS
+PUPPET_GATLING_SCALE_INCREMENT=$PUPPET_GATLING_SCALE_INCREMENT
+PUPPET_GATLING_SCALE_SCENARIO=$PUPPET_GATLING_SCALE_SCENARIO
+ABS_AWS_MOM_SIZE=$ABS_AWS_MOM_SIZE
+============================================"
+}
+
+# Link scale results to nginx content location
+function link_results () {
```

Comment:
With the latest gplt_runner_setup  linking won't be needed as long as the repo is run from $HOME/gatling

Reply:
Updated to use `$HOME/gatling` as the primary working directory.